_sub/security/iam-identity-center/main.tf

```diff
@@ -0,0 +1,31 @@
+data "aws_ssoadmin_instances" "dfds" {}
+
+resource "aws_ssoadmin_permission_set" "capabilityaccess" {
```

Comment:
As we will be associating this with capability AWS accounts, would it make sense to have this permission set's ARN as an output?